README.md

```diff
@@ -5,14 +5,17 @@
 
 # Auth
```

Comment:
I guess we should directly rename the project. Hence I poked you two in #28 to come to a final conclusion regarding the name.